### ISSUE TITLE
y axis label has huge gap between axis label and axis numbers

### ISSUE THREAD

**Issue by jowens** [vega/vega-lite#581]
Minor issue. But that big whitespace is hopefully an easy fix.

<img width="498" alt="vega-lite_editor" src="https://cloud.githubusercontent.com/assets/976970/10118637/51856802-6432-11e5-9cab-e9f63f98df6b.png">

**Comment by jowens** [comment]
```
{
  "marktype": "bar",
  "data": {
    "values": [
      {
        "m_teps": 0.16401416063308716,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 35.45441436767578,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 0.17575806379318237,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 33.5330810546875,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 0.041384510695934296,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 20.627891540527344,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 0.04443415254354477,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 21.458358764648438,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 0.18927541375160217,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 1051.6324462890625,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 0.20337289571762085,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 1023.6077880859375,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 34.21129608154297,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 580.2151489257812,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 35.966224670410156,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 502.16387939453125,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 0.1117696687579155,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 11.019756317138672,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 0.1000901460647583,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 10.410270690917969,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 0.10868960618972778,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 189.5232391357422,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 0.10641517490148544,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 187.35877990722656,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 0.09391451627016068,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 446.43756103515625,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 0.09259837865829468,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 443.1048889160156,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 0.0164876077324152,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 54.624725341796875,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 0.01654699631035328,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 55.096927642822266,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 3071.035400390625,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 3684.061279296875,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 2912.178466796875,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 3530.5595703125,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 0.013857832178473473,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 78.02998352050781,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 0.012905551120638847,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 79.42512512207031,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 1809.6619873046875,
        "parameters": "BFS, directed, no mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 2121.284912109375,
        "parameters": "BFS, undirected, no mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 1724.42333984375,
        "parameters": "BFS, directed, mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 2050.32421875,
        "parameters": "BFS, undirected, mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 0.18537981808185577,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 38.1994743347168,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 0.18337321281433105,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 40.12178421020508,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "ak2010"
      },
      {
        "m_teps": 0.05478702485561371,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 25.023571014404297,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 0.05326380580663681,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 24.77197265625,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "belgium_osm"
      },
      {
        "m_teps": 0.15966972708702087,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 1073.3116455078125,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 0.17343728244304657,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 926.712890625,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "cit-Patents"
      },
      {
        "m_teps": 39.62687683105469,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 651.3740844726562,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 37.61405563354492,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 662.087890625,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "coAuthorsDBLP"
      },
      {
        "m_teps": 0.11892125010490417,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 12.220900535583496,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 0.1219049021601677,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 12.422168731689453,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "delaunay_n13"
      },
      {
        "m_teps": 0.12044372409582138,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 203.1349639892578,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 0.1061120554804802,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 205.55728149414062,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "delaunay_n21"
      },
      {
        "m_teps": 0.09687963873147964,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 481.29412841796875,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 0.08416473120450974,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 476.1145324707031,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "delaunay_n24"
      },
      {
        "m_teps": 0.01447021309286356,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 62.908348083496094,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 0.013993808999657631,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 62.1107292175293,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "europe_osm"
      },
      {
        "m_teps": 5466.40380859375,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 8338.7587890625,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 5170.7744140625,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 8106.59619140625,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "kron_g500-logn21"
      },
      {
        "m_teps": 0.0080620888620615,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 87.52924346923828,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 0.00816779863089323,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 88.382568359375,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "road_usa"
      },
      {
        "m_teps": 1909.693115234375,
        "parameters": "DOBFS, directed, no mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 3440.208984375,
        "parameters": "DOBFS, undirected, no mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 1753.621826171875,
        "parameters": "DOBFS, directed, mark predecessors",
        "dataset": "soc-LiveJournal1"
      },
      {
        "m_teps": 3258.03662109375,
        "parameters": "DOBFS, undirected, mark predecessors",
        "dataset": "soc-LiveJournal1"
      }
    ]
  },
"marktype" : "point",
  "encoding": {
    "y": {
      "scale": {"type": "log"},
      "type": "Q",
      "name": "m_teps",
      "axis": {"title": "MTEPS"}
    },
    "x": {"type": "N","name": "dataset"},
    "color": {"type": "O","name": "parameters"}
  }
}
```

**Comment by kanitw** [comment]
Fixed in PR #710   (Pending review before merging to master)

**Comment by kanitw** [comment]
Note that this will require the upcoming version of Vega.
